api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceService.java

```diff
@@ -33,6 +33,8 @@
 
   List<WorkspaceResponse> getWorkspacesAndPublicWorkspaces();
 
+  WorkspaceResponse getWorkspace(String workspaceNamespace);
```

Comment:
What do you think about either naming this `getWorkspaceOrThrow()` and/or adding a `throws NotFoundException` to the signature. @ericsong and I came to the compromise that doing this on service methods is a net win, even though you ordinarily don't declare runtime exceptions.